shells/go.mod

```diff
@@ -0,0 +1,3 @@
+module manx.go
+
+go 1.16
```

Comment:
should this be 1.13 for parity with gocat?

Reply:
👍  changed it to match